.circleci/config.yml

```
@@ -822,6 +835,24 @@ workflows:
       equal: [profile, << pipeline.parameters.workflow >>]
     jobs:
       - test_profile
+  react-17:
+    triggers:
+      - schedule:
+          cron: '0 12 * * *'
```

```suggestion
          cron: '0 0 * * *'
```

Make this nightly build